app/views/search/search.html.erb

```diff
@@ -42,3 +42,6 @@
     </div>
   <% end %>
 </article>
+<script type="application/javascript">
+  window.history.pushState({},"", "<%= url_for params.except(:authenticity_token, :_method, :controller, :action) %>".replace(/&amp;/g, '&'));
```

Comment:
@matthewford  we rendering search.html.erb, but yeah we can also add params to redirects